src/__tests__/lib/rules/prefer-to-have-value.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * @fileoverview prefer toHaveValue over checking the value attribute
+ * @author Michaël De Boey
+ */
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+import { RuleTester } from "eslint";
+import * as rule from "../../../rules/prefer-to-have-value";
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+new RuleTester().run("prefer-to-have-value", rule, {
+  valid: [
+    "expect(element.value).toBeTruthy()",
```

Comment:
Curious if you can use `.toHaveValue()` with no params for this?

Reply:
I just used `prefer-to-have-attribute` tests and changed everything so it's talking about `toHaveValue` at this moment